src/client/app/containers/import.ts

```diff
@@ -2,11 +2,14 @@ import { Component, ChangeDetectionStrategy } from '@angular/core';
 import * as fromRoot from '../reducers';
 import { Store } from '@ngrx/store';
 import { UI, Snippet, GitHub } from '../actions';
-import { AI, storage } from '../helpers';
+import { UIEffects } from '../effects/ui';
+import { environment, AI, storage, isInsideOfficeApp } from '../helpers';
+import { Request, ResponseTypes } from '../services';
 import { Strings } from '../strings';
 import { isEmpty } from 'lodash';
 
-//strings are specified inline (not imported) for performance reasons
+const viewUrlSettingsPropertyName = 'SnippetToImport';
```

Comment:
const strings usually are in caps such as VIEW_URL_SETTING_KEY. I see mixed styles in this project though.

Reply:
Changed this to all caps.